File: client/src/dto/response/BudgetDetailResponse.ts
import { BudgetGroupStoreTypeResponse } from "./BudgetGroupStoreTypeResponse";
import { BudgetResponse } from "./BudgetResponse";

export interface BudgetDetailResponse {
	budgets: BudgetResponse[];
	priceAsBudgetStoreType: BudgetGroupStoreTypeResponse[];
}

File: service/src/main/java/chivalrous/budgetbuddy/dto/response/BudgetDetailResponse.java
package chivalrous.budgetbuddy.dto.response;

import java.util.List;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
public class BudgetDetailResponse {

	List<BudgetResponse> budgets;
	List<BudgetGroupStoreTypeResponse> priceAsBudgetStoreType;

}
